authorization/invitation/service/invitation_service_blackbox_test.go

```diff
@@ -34,619 +38,693 @@ func (s *invitationServiceBlackBoxTest) SetupTest() {
 	s.invitationRepo = invitationrepo.NewInvitationRepository(s.DB)
 	s.identityRepo = account.NewIdentityRepository(s.DB)
 	s.orgService = s.Application.OrganizationService()
-	s.Application = gormapplication.NewGormDB(s.DB, s.Configuration, factory.WithWITService(&test.DevWITService{}))
+	s.notificationServiceMock = testservice.NewNotificationServiceMock(s.T())
+	s.Application = gormapplication.NewGormDB(s.DB, s.Configuration, factory.WithWITService(&test.DevWITService{}), factory.WithNotificationService(s.notificationServiceMock))
 }
 
-func (s *invitationServiceBlackBoxTest) TestIssueInvitationByIdentityID() {
-	g := s.NewTestGraph()
+func (s *invitationServiceBlackBoxTest) TestIssueInvitation() {
```

Comment:
would you mind adding the following comments in the sub tests:
````
// given
...
// when
...
// then
````
I think these comments greatly help in understanding what data are used (`//given`), what operation is tested (`// when`) and what results are expected (`// then`)